test/Microsoft.AspNetCore.Razor.Runtime.Test/TagHelpers/ReadOnlyTagHelperAttributeListTest.cs

```diff
@@ -128,15 +128,16 @@ public static TheoryData StringIndexerData
         [Theory]
         [MemberData(nameof(StringIndexerData))]
         public void StringIndexer_ReturnsExpectedAttribute(
-            IEnumerable<IReadOnlyTagHelperAttribute> initialAttributes,
+            IEnumerable<TagHelperAttribute> initialAttributes,
             string nameToLookup,
-            IReadOnlyTagHelperAttribute expectedAttribute)
+            TagHelperAttribute expectedAttribute)
         {
             // Arrange
-            var attributes = new ReadOnlyTagHelperAttributeList<IReadOnlyTagHelperAttribute>(initialAttributes);
+            var attributes = new ReadOnlyTagHelperAttributeList(initialAttributes);
+            TagHelperAttribute attribute;
 
             // Act
-            var attribute = attributes[nameToLookup];
+            attributes.TryGetAttribute(nameToLookup, out attribute);
```

Comment:
This was missed by me when I reverted the string indexer changes. Already have tests for `TryGetAttribute`